src/tests/ftest/deployment/agent_failure_isolate.yaml

```diff
@@ -0,0 +1,69 @@
+hosts:
```

Comment:
The config for `agent_failure_isolate.yaml` looks identical to the one for `agent_failure.yaml`, except it has one less server and one more client. Could we just use this config for both, and for `agent_failure`, only run IOR on one client? I think this could reduce the redundant `yaml` and `py` code

Reply:
I merged the two tests to one file. Thanks for pointing out!